frontend/src/pages/sell/Sell.jsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import "./Sell.css";
+import AddItem from "components/add-item/AddItem";
+import SetPrices from "components/set-prices/SetPrices";
+import LocationAndShipping from "components/location-and-shipping/LocationAndShipping";
+import {useState} from "react";
+import { createProduct, uploadProductImages } from "api/Product";
+import moment from "moment/moment";
+import { updateOrCreateCreditCardInfo } from "api/CreditCard";
+import { updateUserAddress } from "api/User";
+import { useNavigate } from "react-router-dom";
+
+function Sell() {
+  const [currentStep, setCurrentStep] = useState(0);
+  const [name, setName] = useState("");
+  const [category, setCategory] = useState(0);
+  const [subCategory, setSubCategory] = useState(0);
+  const [startDate, setStartDate] = useState("");
+  const [endDate, setEndDate] = useState("");
+  const [description, setDescription] = useState("");
+  const [address, setAddress] = useState("");
+  const [country, setCountry] = useState("");
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [nameOnCard, setNameOnCard] = useState("");
+  const [cardNumber, setCardNumber] = useState("");
+  const [expMonth, setExpMonth] = useState(0);
+  const [expYear, setExpYear] = useState(0);
+  const [startingPrice, setStartingPrice] = useState("");
+  const [cvc, setCvc] = useState("");
+  const [files, setFiles] = useState([]);
+  const [creditCardInfo, setCreditCardInfo] = useState(false);
+  const [featured, setFeatured] = useState(0);
+  const [creditCardId, setCreditCardId] = useState(null);
+  const [city, setCity] = useState("");
+  const [zipCode, setZipCode] = useState("");
```

Comment:
This probably should've been a single state object. Don't fix it now, but just for future reference.